File: Back/RecuperApp.Domain/Models/Requests/ShipmentRequest.cs
namespace RecuperApp.Domain.Models.Requests
{
    public class ShipmentRequest
    {
        public int? ShipmentId { get; set; }
        public int CustomerId { get; set; }
        public int EmployeeId { get; set; }
        public int ShipmentTypeId { get; set; }
        public double TotalPrice { get; set; }
        public List<ShipmentDetailRequest> Details { get; set; }
    }
}

File: Front/src/app/models/Requests/ShipmentRequest.ts
import { ShipmentDetailRequest } from './ShipmentDetailRequest';

export class ShipmentRequest {
  Id?: number;
  CustomerId: number = 0;  
  ShipmentTypeId: number = 0;
  TotalPrice: number = 0;
  Details: ShipmentDetailRequest[] = [];
}


